docs/pages/includes/provision-token/github-spec.mdx

```diff
@@ -0,0 +1,61 @@
+```yaml
+kind: token
+version: v2
+metadata:
+  # name identifies the token. When configuring a bot or node to join using this
+  # token, this name should be specified.
+  name: github-token
+spec:
+  # For Machine ID and GitHub joining, roles will always be `Bot` and
+  # join_method will always be `github`.
```

Comment:
```suggestion
  # For Machine ID and GitHub joining, roles will always be "Bot" and
  # join_method will always be "github".
```
Nit: Suggested because the backticks display in YAML comments and the enclosed text is already styled as code